content/docs/command-reference/pull.md

```diff
@@ -107,89 +107,147 @@ reflinks or hardlinks to put it in the workspace without copying. See
 
 ## Examples
 
-For using the `dvc pull` command, a remote storage must be defined. (See
-`dvc remote add`.) For an existing <abbr>project</abbr>, remotes are usually
-already set up and you can use `dvc remote list` to check them. To remember how
-it's done, and set a context for the example, let's define a default SSH remote:
+Let's employ a simple <abbr>workspace</abbr> with some data, code, ML models,
+pipeline stages, such as the <abbr>DVC project</abbr> created for the
+[Get Started](/doc/tutorials/get-started). Then we can see what happens with
+`dvc pull`.
+
+<details>
+
+### Click and expand to setup the project
+
+Start by cloning our example repo if you don't already have it:
 
 ```dvc
-$ dvc remote add -d r1 ssh://_username_@_host_/path/to/dvc/remote/storage
-$ dvc remote list
-r1	ssh://_username_@_host_/path/to/dvc/remote/storage
+$ git clone https://github.com/iterative/example-get-started
+$ cd example-get-started
 ```
 
-> DVC supports several
-> [remote types](/doc/command-reference/remote/add#supported-storage-types).
+</details>
 
-Having some images and other files in remote storage, we can pull all changed
-files from the current Git branch:
+The workspace looks almost like in this
+[pipeline setup](/doc/tutorials/pipelines):
 
 ```dvc
-$ dvc pull --remote r1
+.
+├── data
+│   └── data.xml.dvc
+├── evaluate.dvc
+├── featurize.dvc
+├── prepare.dvc
+├── train.dvc
+└── src
+│   └── ...
+└── train.dvc
+```
+
+We can now just run `dvc pull` to download the most recent `model.pkl`,
+`data.xml`, and other DVC-tracked files into our local <abbr>cache</abbr> and
+<abbr>workspace</abbr>:
+
+```dvc
+$ dvc pull
+
+$ tree example-get-started/
+example-get-started/
+├── data
+│   ├── data.xml
+│   ├── data.xml.dvc
+│   ├── features
+│   │   ├── test.pkl
+│   │   └── train.pkl
+│   └── prepared
+│       ├── test.tsv
+│       └── train.tsv
+├── ...
 ```
 
 We can download specific files that are <abbr>outputs</abbr> of a specific
 DVC-file:
 
 ```dvc
-$ dvc pull data.zip.dvc
+$ dvc pull prepare.dvc
```

Comment:
Maybe use train.dvc since prepare is no longer mentioned above given my suggestions. But it was a nice idea to switch it up here.